NCIOCPL/cgov-digital-platform
# (#3565) Fixed Mega Menu Adapter BaseURL

ODE: https://ncigovcdode508.prod.acquia-sites.com/
Closes #3565

## How do you test this?
1. Go to https://ncigovcdode508.prod.acquia-sites.com/nano
2. Open up dev tools and go to the network tab.
3. Select Fetch/XHR on the types
4. Click on a primary nav item with a mega menu
5. Click on the request for `mega-menu` - this opens up the information about the request 
6. Make sure that the Request URL in the General area has https://ncigovcdode508.prod.acquia-sites.com/nano in the request.
